deploy/helm/sumologic/conf/metrics/otelcol/processors.yaml

```diff
@@ -16,7 +16,7 @@ filter/drop_unnecessary_metrics:
       - resource.attributes["job"] != "pod-annotations" and IsMatch(name, "scrape_.*")
```

Comment:
should we also drop metrics coming in from pod annotations by default?

Reply:
It's breaking change